demo/helm/rook-operator/README.md

```diff
@@ -15,7 +15,7 @@ Alternatively, to deploy from a local checkout of the rook codebase (until the r
 
 ```console
 $ cd demo/helm/rook-operator
-$ helm install --name rook-operator --namespace rook .
+$ helm install --name rook-operator --namespace default .
```

Comment:
no let's leave it in the rook namespace. The rest of the change looks good though.